src/main.rs

```diff
@@ -612,9 +613,9 @@ async fn start_ldk() {
 		}
 	});
 
-	// Step 14: Connect and Disconnect Blocks
+	// Step 15: Connect and Disconnect Blocks
 	if chain_tip.is_none() {
-		chain_tip = Some(init::validate_best_block_header(bitcoind_client.as_ref()).await.unwrap());
+		chain_tip = Some(polled_chain_tip);
 	}
```

Comment:
In another commit, do you mind consolidating this so `chain_tip` is initialized once at step 10 such that an `Option` is not needed?

```rust
let chain_tip = if restarting_node {
    // ...
} else {
    polled_chain_tip
};

Reply:
Implemented in 430084f0e3ec5a05f9521b6ee6cf9ad43f27e8d4